build_test/lib/src/test_bootstrap_builder.dart

```diff
@@ -7,7 +7,7 @@ import 'dart:async';
 import 'package:build/build.dart';
 import 'package:crypto/crypto.dart';
 import 'package:path/path.dart' as p;
-// ignore: deprecated_member_use
+// ignore: deprecated_member_use, import_of_legacy_library_into_null_safe
```

Comment:
We don't want to be ignoring any of these at least not under `lib` - we can push on migrating `test_core/backend.dart` sooner than later, but for now I would opt out this file and any file that imports it.

Otherwise when that library does migrate it could break this library.